frontend/src/cmps/popovers/popover-close-board.jsx

```diff
@@ -0,0 +1,37 @@
+import { useNavigate } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+import { PopoverCmp } from "./popover-cmp";
+
+import { userService } from "../../services/user.service";
+import { onRemoveBoard } from "../../store/actions/board.actions";
+import { closePopover } from "../../store/actions/app.actions";
+
+export const PopoverCloseBoard = ({ elPos, handleClose }) => {
+  const board = useSelector((state) => state.boardReducer.board);
+  const loggedinUser = userService.getLoggedinUser();
+
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const onCloseBoard = (ev) => {
+    ev.preventDefault();
+    if (loggedinUser.fullname === board.createdBy.fullname) {
+      dispatch(onRemoveBoard(board));
```

Comment:
send boardId instead the whole board object

Reply:
Done

Reply:
Current code: 

export function loadBoard(boardId) {
  return async (dispatch) => {
    try {
      const board = await boardService.getById(boardId);
      dispatch({ type: "SET_BOARD", board });
    } catch (err) {
      console.log("board.actions: err @ loadBoard", err);
    }
  };
}